sourcegraph/sourcegraph
# Fix browser extension low quality toolbar icon

Closes https://github.com/sourcegraph/sourcegraph/issues/16218.

### How to test
- `yarn --cwd client/browser build`
- [Reinstall browser extension (Chrome/Firefox/Safari)](https://github.com/sourcegraph/sourcegraph/tree/main/client/browser#development)
- Check the quality of toolbar icon
